src/main.js

```diff
@@ -414,10 +418,32 @@ function handleAppWebContentsCreated(dc, contents) {
   });
 
   contents.on('new-window', (event, url) => {
-    if (isTrustedURL(url)) {
+    event.preventDefault();
+    if (!isTrustedURL(url)) {
+      log.info(`Untrusted popup window blocked: ${url}`);
       return;
     }
-    event.preventDefault();
+    if (popupWindow && popupWindow.getURL() === url) {
+      return;
+    }
+    if (!popupWindow) {
+      popupWindow = new BrowserWindow({
+        parent: mainWindow,
+        alwaysOnTop: true,
```

Comment:
`alwaysOnTop: true` makes it so you cannot click anywhere in MM while the popup is open. Is this necessary for the fix, or is there an alternative?

Reply:
Sorry, are you saying that is what's happening for you or asking if it could? There is a modal option that I didn't use to specifically avoid that. I added the always on top because those popup login windows have a tendency to get lost behind other windows in my experience. 0/5 on keeping it, but happy to remove if you are experiencing this. I can't test anymore as the Jira plugin on community is now giving me the ephemeral post asking me to login to Jira in my browser.

Reply:
> Sorry, are you saying that is what's happening for you or asking if it could?

I am experiencing the issue. I would definitely prefer to have the ability to put the MM app in focus while the popup is open.

> I can't test anymore as the Jira plugin on community is now giving me the ephemeral post asking me to login to Jira in my browser.

The Jira plugin needs to be updated on community to show the popup again

Reply:
Okay, I'll remove that setting. :)